CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 * Remove deprecated `Kafka.spec.topicOperator` classes and deployment logic
 * Use Java 11 as the Java runtime
+* Upgraded to Cruise Control version 2.0.108 - This removes the need to manually create metrics topics if topic auto creation is disabled.
```

Comment:
```suggestion
* Remove the needto manually create Cruise Control topics if topic auto creation is disabled.
```

Reply:
Why is the version of the upstream system not relevant?

Reply:
And why is it relevant? Especially with the weird CC versioning and the version change being patch only, for most people it is meaning less.

Reply:
Well it is a _log_ of changes and this is a change? Maybe the users can decide for themselves if the version number is important info or not? 

If you feel that strongly about it I will take it out. But I don't think we have the CC version documented anywhere (other than the upstream poms) so we should probably add it to the docs somewhere?

Reply:
I feel strongly about it. We do not document there patch versions of Zookeeper, minor versions of Fabric8 or Vert.x updates etc.